### PR TITLE
update version and goldens

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: internal/server/statvar/golden/search_statvar/fem.json [hunk 1]
```diff
@@ -29636,66 +29642,66 @@
     },
     {
       "dcid": "dc/g/Student_EnrollmentLevel-MiddleSchool_Gender-Female_SocialCategory",
-      "name": "Student With Gender = Female, Enrollment Level = MiddleSchool, Social Category",
+      "name": "Student With Gender = Female, Enrollment Level = Middle School, Social Category",
```
Comment (pradh) [inline internal/server/statvar/golden/search_statvar/fem.json:29645]: This seems like a good improvement.

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 2]
```diff
@@ -17208,11 +17208,11 @@
...
         "NetMeasure_Income_Farm",
-        "NumberOfDays_Concentration_PM25_Above12PPM",
-        "NumberOfDays_Concentration_PM25_Above150PPM",
-        "NumberOfDays_Concentration_PM25_Above250PPM",
-        "NumberOfDays_Concentration_PM25_Above35PPM",
-        "NumberOfDays_Concentration_PM25_Above55PPM",
+        "NumberOfDays_Concentration_SmokePM25_Above12PPM",
+        "NumberOfDays_Concentration_SmokePM25_Above150PPM",
+        "NumberOfDays_Concentration_SmokePM25_Above250PPM",
+        "NumberOfDays_Concentration_SmokePM25_Above35PPM",
```
Comment (pradh) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:17214]: These are expected from RFF update

**Review by pradh — APPROVED** [review]
Generally looking good.

**Review by shifucun — APPROVED** [review]
Generally looking good if the stat var changes expected from corresponding import changes

**Pull request merged**